src/js/flow/acquia_lift.variations.js

```diff
@@ -53,18 +53,15 @@
   };
```

Comment:
Also need to look at Drupal.behaviors.acquiaLiftVariations in this file.  It reads either "element" or "page" from toolbarEditMode.  This is set in the acquia_lift.module function acquia_lift_init from the $_SESSION['acquia_lift_element_trigger'] value which is in turn set by acquia_lifty.admin.wizard.inc acquia_lift_personalize_campaign_wizard_variations_submit_element_add function.  Since you no longer check the type and it won't matter anymore... for now it could just be a boolean or an empty check to set whether or not to trigger variation editing.  In the future, this could be the option_id to edit when we get to that point.